### PR TITLE
Add google meet link to email notifications

### PR DESCRIPTION
**Pull request by femyeda**

### CODE REVIEW

File: lib/emails/EventOrganizerMail.ts [hunk 1]
```diff
@@ -18,14 +18,24 @@ export default class EventOrganizerMail extends EventMail {
...
-      description: this.calEvent.description + this.stripHtml(this.getAdditionalBody()) + this.stripHtml(this.getAdditionalFooter()),
-      duration: { minutes: dayjs(this.calEvent.endTime).diff(dayjs(this.calEvent.startTime), 'minute') },
+      description:
+        this.calEvent.description +
+        this.stripHtml(this.getAdditionalBody()) +
+        this.stripHtml(this.getAdditionalFooter()),
+      duration: { minutes: dayjs(this.calEvent.endTime).diff(dayjs(this.calEvent.startTime), "minute") },
       organizer: { name: this.calEvent.organizer.name, email: this.calEvent.organizer.email },
-      attendees: this.calEvent.attendees.map( (attendee: any) => ({ name: attendee.name, email: attendee.email }) ),
+      attendees: this.calEvent.attendees.map((attendee: any) => ({
```
Comment (baileypumfleet) [inline lib/emails/EventOrganizerMail.ts:35]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689305)
```suggestion
      attendees: this.calEvent.attendees.map((attendee: unknown) => ({
```

Comment (baileypumfleet) [inline lib/emails/EventOrganizerMail.ts:35]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689305)
```suggestion
      attendees: this.calEvent.attendees.map((attendee: unknown) => ({
```

**Review by Malte-D — APPROVED** [review]
It works and the code looks good too (After Baileys comments are merged)

File: lib/emails/EventOrganizerMail.ts (hunk 1 above)
Comment (baileypumfleet) [inline lib/emails/EventOrganizerMail.ts:35]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689305)
```suggestion
      attendees: this.calEvent.attendees.map((attendee: unknown) => ({
```

**Pull request merged**